dev_scripts_helpers/github/dockerized_invite_gh_contributors.py

```diff
@@ -1,46 +1,42 @@
 #!/usr/bin/env python
 """
-Invite GitHub collaborators listed in a Google Sheet while obeying the
-50-invite / 24-hour cap.
+Invite GitHub collaborators listed in a Google Sheet/CSV while obeying the
+50-invite / 24-hour cap. This requires some dependencies, which is why it is
+executed in a Docker container.
 
-> automate_collaborator_invitations.py \
+Example (Google Sheet):
+
+> invite_gh_contributors.py \
     --drive_url "https://docs.google.com/spreadsheets/d/1Ez5uRvOgvDMkFc9c6mI21kscTKnpiCSh4UkUh_ifLIw
     /edit?gid=0#gid=0" \
     --gh_token "$GH_PAT" \
```

Comment:
We should not pass secrets via command line, since they are saved in the history.

We should pass them through an env var (e.g., `GITHUB_TOKEN`)